java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/protocol/LspTemplateUI.java

```diff
@@ -138,6 +138,7 @@ private LspTemplateUI() {
                 final DataFolder target = targetAndName.first();
                 final String name = targetAndName.second();
                 Map<String,String> prjParams = new HashMap<>();
+                // PENDING: this is a mavenism; 
                 prjParams.put("version", "1.0-SNAPSHOT"); // NOI18N
```

Comment:
Please modify existing templates (as exposed by VSNetBeans) to use `1.0-SNAPSHOT` as default and remove this attribute.